examples/kubernetes/snapshot/specs/snapshot-import/volume-snapshot-content.yaml

```diff
@@ -8,6 +8,7 @@ spec:
     name: static-snapshot-demo
     namespace: default 
   source:
-    snapshotHandle: snap-0fba4d7649d765c50
+    snapshotHandle: snap-0644009a49add11ac
```

Comment:
Why do we need this?

Reply:
This is not necessary, It is just for my testing.. I will change it back, good catch